recipes/install.rb

```diff
@@ -31,20 +31,6 @@
   only_if { conda_helpers.is_upgrade }
 end
 
-group node['kagent']['userscerts_group'] do
-  action :create
-  gid node['kagent']['userscerts_group_id']
-  not_if "getent group #{node['kagent']['userscerts_group']}"
-  not_if { node['install']['external_users'].casecmp("true") == 0 }
-end
-
-group node['kagent']['userscerts_group'] do
-  action :modify
-  members node['livy']['user']
-  append true
-  not_if { node['install']['external_users'].casecmp("true") == 0 }
-end
-
```

Comment:
Is the plan to also remove the `livy` user from this group during an upgrade?

Reply:
I see your point, but I don't think it's strictly necessary during an upgrade. The group is not used for anything now. So it doesn't do any harm if livy is there.

Reply:
Sure, I agree hence the approval.